xbmc/addons/PVRClient.cpp

```diff
@@ -669,8 +700,8 @@ PVR_ERROR CPVRClient::GetEPGForChannel(const CPVRChannelPtr &channel, CPVREpg *e
   ADDON_HANDLE_STRUCT handle;
   handle.callerAddress  = this;
   handle.dataAddress    = epg;
-  handle.dataIdentifier = bSaveInDb ? 1 : 0; // used by the callback method CAddonCallbacksPVR::PVRTransferEpgEntry()
-  PVR_ERROR retVal = m_struct.GetEpg(&handle,
+  handle.dataIdentifier = bSaveInDb ? 1 : 0; // used by the callback method CPVRClient::PVRTransferEpgEntry()
```

Comment:
The method mentioned in the comment imo should read <code>CPVRClient::cb_transfer_epg_entry</code>